### DISCUSSION TITLE
Add a custom button at the top of the sidebar

### DISCUSSION THREAD

**Discussion by AlmostSuspense** [docsifyjs/docsify#1880] [Q&A] [answered]
Hi there, I'm looking to add a button to the top of the sidebar. I want it to link to an external site, is this something i can do with docsify?

**Comment by trusktr** [comment, marked as answer]
Not directly, but once your app is loaded you can use the [`mounted` hook](https://docsify.js.org/#/write-a-plugin?id=write-a-plugin) to do something after the site is loaded, and you can write some JS that can inject the button.

In the future I hope to add a templating system to make this easier.
